### DISCUSSION TITLE
[Feature] Support setting as Android Cloud Media App

### DISCUSSION THREAD

**Discussion by tylers-username** [immich-app/immich#14710] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

Google accounts now support setting ones preferred cloud media app. It would be great to be able to leverage that with immich.

### Platform

- [ ] Server
- [X] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
#7101
